data/ecosystems/h/hedera.toml

```diff
@@ -1,98 +1,31 @@
 # Ecosystem Level Information
 title = "Hedera"
 
-sub_ecosystems = ["Calaxy"]
-
-github_organizations = ["https://github.com/hashgraph"]
+sub_ecosystems = [
+"Calaxy", 
+"Heliswap", 
+"Hashport",
+"SaucerSwap Labs",
+"Heliswap",
+"Buidler Labs",
+"Blade Labs",
+"Burstall",
+"PieFi Platform",
+"Mintmaster Labs",
+"HashGuild",
+"TxMQ, Inc",
+"HSuite",
+"Trust Enterprises",
+"DOVU",
+]
+
+github_organizations = [
+"https://github.com/hashgraph",
+"https://github.com/limechain",
+"https://github.com/bugbytesinc/",
+]
 
 # Repositories
-[[repo]]
-url = "https://github.com/agear13/hederaweathernfts"
-
-[[repo]]
-url = "https://github.com/AkashWarlocks/notarization-on-blockchain"
-
-[[repo]]
-url = "https://github.com/Blade-Labs/hedera-mirror-node"
-
-[[repo]]
-url = "https://github.com/Blade-Labs/swift-blade"
-
-[[repo]]
-url = "https://github.com/Brayton-Lordianto/AnimalLoversHacks"
-
-[[repo]]
-url = "https://github.com/bugbytesinc/Hashgraph"
-
-[[repo]]
```

Comment:
Any reason to remove all of these?  Most of them still exist, so we keep even older repos around for posterity reasons.

Reply:
Ah, no reason to remove these — will add back in.